src/renderer/utils/equalizerApi.ts

```diff
@@ -117,6 +117,16 @@ export const savePreset = (preset_name: string): Promise<void> => {
   return promisifyResult(setterResponseHandler, channel);
 };
 
+/**
+ * Delete a preset file in preset folder
+ * @returns { Promise<void> } if delete was succesfull
+ */
+export const deletePreset = (preset_name: string): Promise<void> => {
```

Comment:
sorry, minor thing, we've been using camel case for variable names for the most part - could we try to be more consistent with that?

Reply:
fixed